.circleci/src/jobs/promote_ios.yml

```diff
@@ -0,0 +1,20 @@
+# Promotes the app from Testflight to the Apple App Store.
+parameters:
+  build_config_name:
+    type: enum
+    enum: [bayern, nuernberg]
+    default: bayern
+macos:
+  xcode: 15.0.1
```

Comment:
we should use xcode 15.2.0 as we do for all other jobs

Reply:
PR is older than this updates, but will adjust it.